src/Compilers/CSharp/Portable/Symbols/MemberSymbolExtensions.cs

```diff
@@ -453,48 +453,6 @@ internal static ImmutableArray<Symbol> GetExplicitInterfaceImplementations(this
             }
         }
 
-        internal static TypeSymbol GetTypeOrReturnType(this Symbol member)
```

Comment:
Are you referring to the rename of the "this" parameter, which was and still is a Symbol? This is applicable to many other methods in this file. Why stop at this two?

---
In reply to: [80815690](https://github.com/dotnet/roslyn/pull/13999#discussion_r80815690) [](ancestors = 80815690,80807640)